docs/interactions/Message_Components.md

```diff
@@ -14,20 +14,25 @@ The top-level `components` field is an array of [Action Row](#DOCS_INTERACTIONS_
+| options?     | array of [select options](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menu-object-select-option-structure) | the choices in the select, max 25                                                         | [Select Menus](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menus)                                                                   |
+| placeholder? | string                                                                                                      | custom placeholder text if nothing is selected, max 100 characters                        | [Select Menus](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menus), [Text Inputs](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/text-inputs) |
+| min_values?  | integer                                                                                                     | the minimum number of items that must be chosen; default 1, min 0, max 25                 | [Select Menus](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menus)                                                                   |
+| max_values?  | integer                                                                                                     | the maximum number of items that can be chosen; default 1, max 25                         | [Select Menus](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menus)                                                                   |
+| components?  | array of [components](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/component-object)                               | a list of child components                                                                | [Action Rows](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/action-rows)                                                                     |
+| label?       | string                                                                                                      | the label for this component                                                              | [Text Inputs](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/text-inputs)                                                                     |
+| min_length?  | integer                                                                                                     | the minimum input length for a text input                                                 | [Text Inputs](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/text-inputs)                                                                     |
+| max_length?  | integer                                                                                                     | the maximum input length for a text input                                                 | [Text Inputs](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/text-inputs)                                                                     |
```

Comment:
Is there a max length regardless if one is set? It would be pretty bad if someone could send huge amounts of text

Reply:
it's 4000

Reply:
Cool. So include that in the documentation.